lib/Net/SAML2/SP.pm

```diff
@@ -277,7 +277,7 @@ sub _build_cert_text {
     return $text;
 }
 
-=head2 authn_request( $destination, $nameid_format )
+=head2 authn_request( $destination, $nameid_format, $force_authn, $is_passive )
```

Comment:
As a question: `authn_request($dest, %options)` so we can add more (optional) options to the authnrequest? We could also go for `authn_request($dest, $format, %options)`. But I'm not sure we should expand the API of this sub too much, because when you only want `is_passive`, you now need to supply `force_authn` and it might be harder to read: `authn_request($dest, $format, undef, 1, more, options, here, in the, future)`

Reply:
Yeah, I was not happy with this.  In fact I suspect that the SP and probably IdP interfaces have been forgotten in the past and may have bugs

Reply:
we could as you mention on IRC add things as %options.  Really only $destination is required the nameid_format is often undef